vars/seleniumExecuteTests.groovy

```diff
@@ -116,7 +120,13 @@ void call(Map parameters = [:], Closure body) {
                 config.stashContent = config.testRepository
                     ?[GitUtils.handleTestRepository(this, config)]
                     :utils.unstashAll(config.stashContent)
-                body()
+                if (config.seleniumHubCredentialsId) {
+                    withCredentials([usernamePassword(credentialsId: config.seleniumHubCredentialsId, passwordVariable: 'PIPER_SELENIUM_GRID_PASSWORD', usernameVariable: 'PIPER_SELENIUM_GRID_USER')]) {
```

Comment:
We should name all three selenium grid or selenium hub.

Reply:
Good catch, I'll fix it...